.github/workflows/build-and-lint-front.yml

```diff
@@ -13,12 +13,20 @@ permissions:
   checks: write
 
 jobs:
+  check-package-lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Node Dependencies
+        uses: ./.github/actions/setup-node-deps
```

Comment:
If you setup nodejs dep before the check, I'm afraid the package-lock will always be in sync right ?

For this job I think we don't want to use `setup-node-deps` at all